src/test/browser.js

```diff
@@ -2,35 +2,36 @@
 
 const Server = require('karma').Server
 
-const getConfig = require('./browser-config')
 const utils = require('../utils')
 
-function karma (config) {
-  return new Promise((resolve, reject) => {
-    const server = new Server(config, (exitCode) => {
-      if (exitCode > 0) {
-        reject(new Error('Some tests are failing'))
-      }
+module.exports = (getConfig) => {
+  function karma (config) {
```

Comment:
Please don't nest `function` definitions inside another function. Also not sure why this was done in the first place.